### PR TITLE
Download CSV

### PR DESCRIPTION
**Pull request by srleach**
## Features
- Added missing fields to CSV Application Export

[Delivers#94946948]

### CODE REVIEW

File: app/Basket/Application.php [hunk 1]
```diff
@@ -150,4 +155,37 @@ public function location()
         return $this->belongsTo('App\Basket\Location');
     }
 
+    /**
+     * Get an Export Safe version of the model to generate a CSV/JSON Export.
+     */
+    public function getExportableFields(){
+        $exportFields = [
```
Comment (wnowicki) [inline app/Basket/Application.php]: minor thing, as we are not doing anything else with this var, please just return it `return [ ...`

File: app/ExportableModelInterface.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * Interface ExportableModelInterface
+ * @author SL
+ * @package App
+ */
+interface ExportableModelInterface
+{
+    /**
+     * @return mixed
```
Comment (wnowicki) [inline app/ExportableModelInterface.php]: `mixed` really? ;)

**Comment by wnowicki** [comment]
Good! Please correct those minor thing and we can merge it then.

File: app/ExportableModelInterface.php (hunk 2 above)
Comment (srleach) [inline app/ExportableModelInterface.php]: This was PHPStorm ;) I'll check & correct now

**Pull request merged**